.vib/opencart/cypress/cypress.json

```
@@ -0,0 +1,12 @@
+{
+  "env": {
+    "username": "user",
+    "password": "ComplicatedPassword123!4",
+    "smtpHost": "smtptesthost",
+    "smtpUser": "smtpUser"
+  },
+  "hosts": {
+    "vmware-opencart.my": "127.0.0.1"
```

This might work, but we should use the actual IP assigned to the service: 
```suggestion
    "vmware-opencart.my": "{{ TARGET_IP }}"
```

---

Yes indeed. I missed the step of removing this and replacing it with an adequate variable.